src/js/themes/base.d.ts

```diff
@@ -1125,6 +1125,14 @@ export interface ThemeType {
     container?: BoxProps;
     label?: TextProps;
   };
+  spinner?: {
+    container?:
+      | BoxProps
+      | { color?: ColorType }
+      | { size?: 'xsmall' | 'small' | 'medium' | 'large' | 'xlarge' | string };
+    icon?: any;
```

Comment:
Should this be `JSX.Element`, like Button.icon?

Reply:
Almost. Button.icon is a prop, so it will make sense to use a JSX for it e.g.`<Button icon={<Add />} />`. 
The theme is looking for a React Node and not necessarily a JSX element:
```
spinner: {
    icon: Add
}
```

The type that worked best was React.ReactNode. I moved the ThemedIcon story from 'js' to 'tsx' to verify it.